crypto-ffi/bindings/jvm/build.gradle.kts

```diff
@@ -10,6 +10,18 @@ java {
     targetCompatibility = JavaVersion.VERSION_17
 }
 
+dokka {
+    moduleName.set("CoreCrypto")
+    pluginsConfiguration.html {
+        footerMessage.set("Copyright Wire GmbH")
+    }
+    dokkaSourceSets.main {
+        sourceLink {
+            remoteUrl("https://github.com/wireapp/core-crypto/tree/main/crypto-ffi/bindings/jvm")
+        }
+    }
+}
+
```

Comment:
Can we move this and the same block in `android` to toplevel `bindings/build.gradle.kts`?
They look identical, and that way we could define this in only one place.

Reply:
So I explored this. First, no we can't just move the block to `bindings/build.gradle.kts` but what they suggest in the migration guide is to create convention plugin which applies the dokka configuration to all relevant modules.

https://kotlinlang.org/docs/dokka-migration.html#multi-module-projects-without-convention-plugins

Unfortunately when applying the dokka configuration in this way it breaks the setup and no documentation is generated at all :/

Reply:
Oh well, nothing is easy with gradle...

Reply:
Oh wait, just tried some obvious which seems to work.